### ISSUE TITLE
Questions about the hyperparameters in 'config' 

### ISSUE THREAD

**Issue by ihatedebug** [EleutherAI/gpt-neo#166]
I'm interested in training GPT-like models, and would like to ask about the hyperparameters that is written in the 'configs' folder.

1. About 'gpt3_XL_256_Pile.json' and 'gpt3_13B_256_Pile.json', are the hyperparameters actually the same values as the ones that is succeeded in pre-train? 
2. About the other json model configs, are the hyper parameters (e.g, learning rate and warm up method) the best values you've been found, using openwebtext2 dataset? Some values are not consistent with the ones suggested in GPT-3 paper.
3. Do pre-trained GPT3_1.3B and GPT3_2.7B use sparse attention method? or it's just scale-up of GPT2? 

Thanks.

**Comment by sdtblck** [comment]
1. Our config files are a bit of a mess currently and are the ones we used for development. I'll be cleaning this up in a PR in the next few days.
2. Since there's many different hparams I can't comment on whether they're 'the best we've found' or not. But in general i think the main differences are the regularization. We've found dropout and weight decay only hinder training at these scales, as well as slowing it down, so set them both to 0 across the board.
3.  Our pretrained models use local attention, which is different to sparse, but not exactly the same as gpt2.
